.github/workflows/meilisearch-prototype-tests.yml

```diff
@@ -1,26 +1,34 @@
 # Testing the code base against a specific Meilisearch feature
-name: Beta tests
+name: Meilisearch prototype tests
 
 # Will only run for PRs and pushes to *-beta
 on:
-  push:
-    branches: ['**-beta', '!bump-meilisearch-v[0-9]*.[0-9]*.[0-9]*-beta']
   pull_request:
-    branches: ['**-beta', '!bump-meilisearch-v[0-9]*.[0-9]*.[0-9]*-beta']
+    branches:
+      - 'prototype-beta/**'
+  push:
+    branches:
+      - 'prototype-beta/**'
 
 jobs:
   meilisearch-version:
     runs-on: ubuntu-latest
     outputs:
-      version: ${{ steps.grep-step.outputs.version }}
+      version: ${{ steps.grep-step.outputs.meilisearch_version }}
     steps:
       - uses: actions/checkout@v3
       - name: Grep docker beta version of Meilisearch
         id: grep-step
         run: |
-          MEILISEARCH_VERSION=$(sh .github/scripts/beta-docker-version.sh)
-          echo $MEILISEARCH_VERSION
-          echo ::set-output name=version::$MEILISEARCH_VERSION
+          MEILISEARCH_VERSION=$(sh .github/scripts/prototype-docker-version.sh $GITHUB_REF_NAME)
+          if [ -z "$MEILISEARCH_VERSION" ]
+          then
+            echo "Could not find any docker image for this prototype"
```

Comment:
```suggestion
            echo "Could not find any docker image for this prototype $GITHUB_REF_NAME"
```

Reply:
Always give as many information as possible in error message, the future of you will thank you